blender/arm/material/make_cluster.py

```diff
@@ -75,7 +80,7 @@ def write(vert: shader.Shader, frag: shader.Shader):
     frag.write('    f0')
 
     if is_shadows:
-        frag.write('\t, li, lightsArray[li * 3 + 2].x, lightsArray[li * 3 + 2].z != 0.0') # bias
+        frag.write('\t, li, lightsArray[li * 3 + 2].x, lightsArray[li * 3 + 2].z != 0.0, opacity != 1.0') # bias

```

Comment:
Here in forward we don't know if the material is opaque or not without using opacity.